### ISSUE TITLE
Latest changes to the rules

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#110]
**Today we will be teaching the changes we have made based on the rules**

- Today we added a fan, changed the programming of the free track and started programming the track with obstacles.

**Comment by WROMagnet** [comment]
1- We have added a fan just below the DC motor of our vehicle. We have integrated it because the vehicle's motor was getting very hot because when adding the Lego Spike motors and the controller the vehicle has more weight which means the motor has to make more effort and it gets hot. So that the DC motor does not get damaged by getting too hot we have decided to add a fan to cool the DC motor.

https://github.com/user-attachments/assets/08e41f6a-c990-4eba-bf09-76c398911fb1

-  The fan is connected directly to the voltage coming out of the power switch and the negative is connected to the negative of the Arduino Mega.

**Comment by WROMagnet** [comment]
2- We changed the programming of the free track a little because the rules of the future engineers category tell us that if the vehicle has more than one driver, the drivers must start their programming at the same time.

- We added these programming lines :

`For controllers to start programming at the same time`

![Image](https://github.com/user-attachments/assets/6943b86b-fda0-47f3-9836-dbf0f104e9c4)

![Image](https://github.com/user-attachments/assets/748bb4a3-8d5b-4e75-a44f-734ef6f9869f)

- These programming lines make it so that when the Arduino Mega detects 2 pulses from the Lego Spike at the same time it starts programming. The Lego Spike has a programming that once it starts it sends two pulses which is what makes the Arduino Mega also start.

`To stop programming once the vehicle makes 3 turns.`

![Image](https://github.com/user-attachments/assets/ae27ccda-4df1-459a-828e-81009c4a29b2)

![Image](https://github.com/user-attachments/assets/6f6e93ce-a83c-4f91-b9fd-bf5be5ee8041)

- These programming lines make it so that when a pulse is received on a specific pin, the vehicle stops. The Lego Spike with the color sensor that pulses the ground when it detects the blue color 12 times sends a pulse which the Arduino Mega with this command when it detects it stops the vehicle.
